app/pages/resource/reservation-calendar/ReservationCalendarContainer.js

```diff
@@ -77,10 +81,41 @@ export class UnconnectedReservationCalendarContainer extends Component {
 
     return (
       <div className="reservation-calendar">
-        <Calendar
-          date={date}
-          onChange={this.onDateChange}
-        />
+        <div className="calendar-availability">
```

Comment:
- [ ] The ResourceAvailabilityCalendar should be a component/container of it's own. Optimally it would just be given the resource/resourceId and maybe selected date + onDateChange function but calculating the availability would not be the concern of the component using the calendar.